docs/installation/installing.rst

```diff
@@ -52,12 +52,22 @@ Tip
         listeners=http://localhost:8088
         ui.enabled=true
 
+    You can also set or override any of the KSQL server properties by specifying them in the ``KSQL_OPTS`` environment variable.
+    The properties are standard java system properties, for example, ``KSQL_OPTS=-Dbootstrap.servers=myhost:9090``
+    Any properties set in this way take precedence over those specified in the ``ksql-server.properties`` file. This allows you to deploy a
+    common set of properties and override specific properties as needed.
+
 #.  Start a server node with this command:
 
     .. code:: bash
 
         $ <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
 
+    or with overriding properties:
+
+    .. code:: bash
+        $ KSQL_OPTS=-Dui.enabled=false <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
```

Comment:
@dguy fyi, a line break is required between `.. code:: bash` and the code sample. e.g. should look like

```
.. code:: bash

       $ KSQL_OPTS=-Dui.enabled=false <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
```